vendor/github.com/go-openapi/spec/.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
```

Comment:
I'm curious why we're importing others' CI configuration? It it possible to exclude such files?

Reply:
I guess because godeps includes them. I don't think it's worth the complexity to exclude them explicitly with some script.